packages/graphql/src/resolvers/attestations/checkTelegramStatus.js

```diff
@@ -0,0 +1,63 @@
+import validator from '@origin/validator'
+import get from 'lodash/get'
+
+import contracts from '../../contracts'
+
+import sleep from '../../utils/sleep'
+
+const MAX_TRIES = 20
+
+async function checkTelegramStatus(_, { identity }) {
+  const bridgeServer = contracts.config.bridge
+  if (!bridgeServer) {
+    return { success: false, reason: 'No bridge server configured' }
+  }
+
+  let tries = 0
+  const url = `${bridgeServer}/api/attestations/telegram/status?identity=${identity}`
+
+  while (tries < MAX_TRIES) {
+    const response = await fetch(url, {
+      headers: { 'content-type': 'application/json' },
```

Comment:
nit: this is a GET, right ? So there is no JSON payload and this header is not necessary.

Reply:
Yeah, will remove it